api/src/main/java/org/openmrs/module/ModuleFactory.java

```diff
@@ -704,12 +704,8 @@ public static Module startModuleInternal(Module module, boolean isOpenmrsStartup
 				for (Extension ext : module.getExtensions()) {
 					
 					String extId = ext.getExtensionId();
-					List<Extension> tmpExtensions = moduleExtensionMap.get(extId);
-					if (tmpExtensions == null) {
-						tmpExtensions = new ArrayList<>();
-						moduleExtensionMap.put(extId, tmpExtensions);
-					}
-					
```

Comment:
Did you intentionally replace ArrayList with Vector?

Reply:
Fixed :)